packages/gogocode-plugin-prettier/index.js

```diff
@@ -24,12 +26,29 @@ module.exports = function (fileInfo, api, options) {
         return sourceCode;
     }
 
-    return prettier.format(sourceCode, {
+    let prettierConfig = {
         trailingComma: 'es5',
         tabWidth: 2,
         semi: false,
         singleQuote: true,
         printWidth: 80,
         parser: /\.vue$/.test(fileInfo.path) ? 'vue' : 'typescript',
-    });
+    };
+    const { prettierrc, rootPath } = options
```

Comment:
for the command `gogocode -s dir/input.js -t gogocode-plugin-prettier -o dir/output.js -p prettierrc=.prettierrc.json`, rootPath value here would be `dir` instead of current working directory.

Probably `options.pwdPath` makes more sense?